modules/terraform/output.go

```diff
@@ -49,6 +49,42 @@ func OutputRequiredE(t testing.TestingT, options *Options, key string) (string,
 	return out, nil
 }
 
+// OutputListOfMaps calls terraform output for the given variable and returns its value as a list of maps.
+// If the output value is not a list of maps, then it fails the test.
+func OutputListOfMaps(t testing.TestingT, options *Options, key string) []map[string]string {
```

Comment:
Thanks for the PR! Once concern is that this only handles a list of maps where all the values are strings. What about maps where the values could be numbers, booleans, or other nested structures? 

I'd like to avoid having to add a never-ending list of `OutputListOfIntMaps`, `OutputListOfBoolMaps`, `OutputListOfMapMaps`, etc. functions. 

Perhaps this one should return `[]map[string]interface{}` instead?